Flepper.QueryBuilder/Commands/Extensions/UpdateCommandExtensions.cs

```diff
@@ -2,14 +2,38 @@
 
 namespace Flepper.QueryBuilder
 {
+    /// <summary>
+    /// Update Command Extension
+    /// </summary>
     public static class UpdateCommandExtensions
     {
+        /// <summary>
+        /// Add Set to query
+        /// </summary>
+        /// <param name="updateCommand">Update Command instance</param>
+        /// <param name="column">Column name</param>
```

Comment:
How about **Column to update**?

Reply:
Yes! For exemple: UPDATE Table SET Column = Value.

We have a Table name a Column name that will be updated and a new value to update.